src/State/CommandState.php

```diff
@@ -64,9 +64,11 @@ public function __construct(
         public ?Artisan $artisan = null,
         public ?string $name = null,
         public NullUserProvider $user = new NullUserProvider,
+        public string $exceptionPreview = '',
     ) {
         $this->deploy = Str::tinyText($this->deploy);
         $this->server = Str::tinyText($this->server);
+        $this->exceptionPreview = Str::text($this->exceptionPreview);
```

Comment:
Confusingly, we don't need this here.

The trimming here is done to make it easier to not have to trim the deploy and server everywhere else.

We already trim the exception preview when we assign it to the parent execution, which is the correct place to do it.

```suggestion
```